modules/redux/src/phenyl-redux.ts

```diff
@@ -1,31 +1,30 @@
-import {
-  AuthCommandMapOf,
-  LocalState,
-  GeneralAction,
-  GeneralTypeMap,
-  ReqResEntityMapOf
-} from "@phenyl/interfaces";
-import { Middleware, Dispatch } from "redux";
-import { MiddlewareCreator, MiddlewareOptions } from "./middleware";
-import { PhenylReduxModule } from "./phenyl-redux-module";
+import { GeneralTypeMap } from "@phenyl/interfaces";
+import { createMiddleware, MiddlewareOptions } from "./middleware";
+import { createReducer, PhenylReducerFunction } from "./reducer";
+import { ActionCreator } from "./action-creator";
 
+export function useRedux<TM extends GeneralTypeMap>(
```

Comment:
[imo] I feel like the name should be `usePhenylRedux`, since, `useRedux` already exists in React. It can cause confusion and Unwanted autoImports